packages/drafts-realm/task-form.gts

```diff
@@ -0,0 +1,315 @@
+    { name: 'Other' },
+  ] as Array<CategorySignature>;
+
+  @action updateSubject(type: { name: string }) {
+    this.selectedSubject = type;
+    this.args.model.title = type.name;
+  }
+
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+
+      <BoxelSelect
+        @searchEnabled={{true}}
+        @searchField='name'
+        @placeholder={{this.subjectPlaceholder}}
+        @selected={{this.selectedSubject}}
+        @onChange={{this.updateSubject}}
+        @options={{this.subjectOptions}}
+        class='select'
+        as |item|
+      >
+        <div>{{item.name}}</div>
+      </BoxelSelect>
+
+    </CardContainer>
+
+    <style>
+      .select {
+        padding: var(--boxel-sp-xs);
+        background-color: white;
+      }
+    </style>
+  </template>
+}
+
+class EmbeddedSecForSubjectField extends Component<typeof SubjectField> {
+  <template>
+    <div class='subject'>{{this.args.model.title}}</div>
+
+    <style>
+      .subject {
+        margin: 0px;
+      }
+    </style>
+  </template>
+}
+
+class SubjectField extends FieldDef {
+  @field title = contains(StringField, {
+    description: `Selected Subject`,
+  });
+
+  static edit = EditSecForSubjectField;
+  static embedded = EmbeddedSecForSubjectField;
+}
+
+/* Task Form */
+class IsolatedSecForTaskForm extends Component<typeof TaskForm> {
+  <template>
+    <div class='card-container'>
+      <div>
+        <h2><@fields.subject /></h2>
+        <div class='comments-box'>
+          <@fields.comments />
+        </div>
+      </div>
+
+      <div class='field-input'>
+        <label>Due Date: </label>
+        <@fields.dueDate />
+      </div>
+
+      <div class='field-input'>
+        <label>Related To: </label>
+        <@fields.relatedTo />
+      </div>
+    </div>
+
+    <style>
+      .card-container {
+        padding: var(--boxel-sp-lg);
+        display: grid;
+        gap: var(--boxel-sp-lg);
+      }
+      .comments-box {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        padding: var(--boxel-sp);
+        background-color: #eeeeee50;
+        border: 1px solid var(--boxel-300);
+        border-radius: var(--boxel-border-radius-xl);
+        margin-top: 1rem;
+      }
+      .field-input {
+        display: flex;
+        gap: var(--boxel-sp-sm);
+        font-size: 0.795rem;
+        flex-wrap: wrap;
+      }
+      .field-input > label {
+        font-weight: 700;
+      }
+      h2 {
+        margin: 0px;
+      }
+    </style>
+  </template>
+}
+
+class ViewSecForTaskForm extends Component<typeof TaskForm> {
+  <template>
+    <div class='card-container'>
+      <h2><@fields.subject.title /></h2>
+
+      <div class='details-container'>
+        <div class='details-heading'>Details</div>
+        <div class='details-content'>
+          <FieldContainer @tag='label' @label='Related To' @vertical={{false}}>
+            <@fields.relatedTo.account />
```

Comment:
Once you change the relatedTo to linksTo. You might need to change this. I think there might be some examples some where